en/mkdocs.yml

```diff
@@ -45,13 +45,10 @@ theme:
 # Navigation
 nav:
   - 'WSO2 Security and Compliance Documentation': 'index.md'
-  - 'Security Processes and Programs':
-    - '': 'security-processes-programs/index.md'
-    - 'Secure Software Development Process': 'security-processes-programs/secure-software-development-process.md'
-    - 'Vulnerability Management Process': 'security-processes-programs/vulnerability-management-process.md'
-    - 'Reward and Acknowledgement Program':
-      - 'Reward and Acknowledgement Program': 'security-processes-programs/reward-and-acknowledgement-program/index.md'
-      - 'Security Hall of Fame': 'security-processes-programs/reward-and-acknowledgement-program/hall-of-fame.md'
+  - 'Security Processes':
+    - '': 'security-processes/index.md'
+    - 'Secure Software Development Process': 'security-processes/secure-software-development-process.md'
+    - 'Vulnerability Management Process': 'security-processes/vulnerability-management-process.md'
```

Comment:
Vulnerability reporting guidelines should also be moved under `Security Reporting`

Reply:
Fixed in 4cf2f78e9fde97193b615b7bda53de23d579955c